corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,7 +1,7 @@
 from corehq.util.elastic import es_index
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-DOMAIN_INDEX = es_index("hqdomains_20160531_1131")
+DOMAIN_INDEX = es_index("hqdomains_20160622_0645")
```

Comment:
Thanks for the heads up, looks like simon fixed it (https://github.com/dimagi/commcare-hq/pull/12208) but sorry for the trouble!